# The median

## Problem 3-25
The following MINITAB output shows the time spent per week searching on line for $30$ Internet users as well as the median of the $30$ times. Verify the median. Would you say this average is typical of the $30$ times? Compare your results with those found in Problem [3.8](02%20The%20arithmetic%20mean.ipynb#Problem-3-8).

In [1]:
p3.25.X <- c( 3,  4,  4,  5,  5,  5,  5,  5,  5,  6,
              6,  6,  6,  7,  7,  7,  7,  7,  8,  8,
              9, 10, 10, 10, 10, 10, 10, 12, 55, 60)

In [2]:
median(p3.25.X)

[1] 7

### Solution 3-25
Note that the two middle values are both $7$ and the mean of the two middle values is $7$. The mean was found to be $10.4$ hours in Problem [3.8](02%20The%20arithmetic%20mean.ipynb#Problem-3-8). The median is more typical of the times than the meain.

---

## Problem 3-26
The number of ATM transactions per day were recorded at $15$ locations in a large city. The data were: $35, 49, 225, 50, 30, 65, 40, 55, 52, 76, 48, 325, 47, 32,$ and $60$. FInd (a) the median number of transactions and (b) the mean number of transactions.

### Solution 3-26
(a) Arranged in order, the data are: $30, 32, 35, 40, 47, 48, 49, 50, 52, 55, 60, 65, 76, 225,$ and $325$. Since there is an odd number of items, there is only one middle value, 50 which is the required median.

(b) The sum of the $15$ values is $1189$. The mean is $1189/15=79.267$.

Note that the median is not affected by the two extreme values $225$ and $325$, while the mean is affected by it. In this case, the median gives a better indication of the average number of daily ATM transactions.

In [3]:
p3.26.X <- c( 35, 49, 225, 50, 30, 65, 40, 55, 52, 76, 48, 325, 47, 32, 60)
cat("(a) Median: ", median(p3.26.X), "\n")
cat("(b) Mean: ", mean(p3.26.X), "\n")

(a) Median:  50 
(b) Mean:  79.26667 


---

## Problem 3-27
If (a) $85$ and (b) $150$ numbers are arranged in an array, how would you find the median of the numbers?

### Solution 3-27
(a) Since there are $85$ items, an odd number, there is only one middle value with $42$ numbers below and $42$ numbers above it. Thus the median is the 43rd number in the aray.

(b) Since there are $150$ items, an even number, there are two middle values with $74$ numbers below them and $74$ numbers above them. The two middle values are the 75th and 76th numbers in the array, and their arithmetic mean is the required median.

---

## Problem 3-28
From Problem [2.8](../../02%20Chapter%202%20Frequency%20Distributions/02%20Solved%20Problems/02%20Frequency%20Distributions,%20Histograms,%20and%20Frequency%20Polygons.ipynb#Problem-2-8), find the median weight of the $40$ male college students at State University by using (a) the frequency distribution of Table [2.8](../../02%20Chapter%202%20Frequency%20Distributions/02%20Solved%20Problems/02%20Frequency%20Distributions,%20Histograms,%20and%20Frequency%20Polygons.ipynb#Table-2-8) (reproduced here as Table [3.7](#Table-3-7)) and (b) the original data.

### Solution 3-28
<a id='Solution-3-28-a-first'></a>
(a) __First method__ (using interpolation)

The weights in the frequency distribution of Table [3.7](#Table-3-7) are assumed to be continuously distributed. In such case the median is that weight for which half the total frequency $(40/2=20)$ lies above it and half lies below it.

In [6]:
Table.3.7 <- matrix(c(3, 5, 9, 12, 5, 4, 2), nrow=7)
rownames(Table.3.7) <- c("118-126", "127-135", "136-144", "145-153", "154-162", "163-171", "172-180")
colnames(Table.3.7) <- "Frequency"

### Table 3-7

In [7]:
Table.3.7

,Frequency
118-126,3
127-135,5
136-144,9
145-153,12
154-162,5
163-171,4
172-180,2


Now the sum of the first three class frequencies is $3+5+9=17$. Thus to give the desired $20$, we require three more of the $12$ cases in the fourth class. Since the fourth class interval, 145-153, actually corresponds to weights $144.5$ to $153.5$, the median must lie $3/12$ of the way between $144.5$ and $153.5$; that is, the median is

\begin{align}
144.5+\frac{3}{12}(153.5-144.5)=144.5+\frac{3}{12}(9)=146.8\text{ lb}
\end{align}

<a id='Solution-3-28-a-second'></a>
__Second method__ (using formula)

Since the sum of the first three and firt four class frequencies are $3+5+9=17$ and $3+5+9+12=29$, respectively, it is clear that the median lies in the fourth class, which is therefore the median class. Then

\begin{align}
L_1=\text{lower class boundary of median class}=144.5 \\
N=\text{number of items in the data}=40 \\
(\sum{f}_1=\text{sum of all classes lower than the median class}=3+5+9=17 \\
f_{\text{median}}=\text{frequency of median class}=12 \\
c=\text{size of median class interval}=9
\end{align}

and thus

\begin{align}
\text{Median}=L_1+\left(\frac{N/2-(\sum{f})_1}{f_{\text{median}}}\right)c=144.5+\left(\frac{40/2-17}{12}\right)(9)=146.8\text{ lb}
\end{align}

(b) Arranged in an array, the original weights are

\begin{align}
119, 125, 126, 128, 132, 135, 135, 135, 136, 138, 138, 140, 140, 142, 142, 144, 144, 145, 145, 146, \\
146, 147, 147, 148, 149, 150, 150, 152, 153, 154, 156, 157, 158, 161, 163, 164, 165, 168, 173, 176
\end{align}

The median is the arithmetic mean of the 20th and 21st weights in this array and is equal to 146 lb.

## Problem 3-29
Figure [3.3](#Figure-3-3) gives the stem-and-leaf display for the number of 2005 alcohol-related traffic deaths for the $50$ states and Washington D.C.  
Stem-and-Leaf Display: Deaths

In [12]:
p3.29.X <- c( 20, 20, 30, 30, 40, 50, 50, 60, 60, 60, 70, 80, 80, 90, 110, 120, 120, 120,
              150, 150, 170, 170, 180, 200, 230, 230, 240, 260, 280, 290, 310, 320, 340,
              360, 370, 390, 420, 420, 460, 460, 490, 500, 510, 520, 540, 540, 580, 630,
              1470, 1560, 1710)

### Figure 3-3

In [13]:
stem(p3.29.X, scale=2)


  The decimal point is 2 digit(s) to the right of the |

   0 | 22334556667889
   1 | 122255778
   2 | 0334689
   3 | 124679
   4 | 22669
   5 | 012448
   6 | 3
   7 | 
   8 | 
   9 | 
  10 | 
  11 | 
  12 | 
  13 | 
  14 | 7
  15 | 6
  16 | 
  17 | 1



Find the mean, median, and mode of the alcohol-related deaths in Fig. [3-3](#Figure-3-3).

### Solution 3-29
The number of deaths range from $20$ to $1710$. The distribution is bimodal. The two modes are $60$ and $120$, both of which occur three times.

The class $(7) 2 \ \ \ 0334689$ is the median class. That is, the median is contained in this class. The median is the middle of the data of the 26th value in the ordered array. The 24th value is $200$, the 25th value is $230$, and the 26th value is also $230$. Therefore the median is $230$.

The sum of the $51$ values is $16,660$ and the mean is $16,660/51=326.67$.

---

## Problem 3-30
Find the median wage of the $65$ employees at the P&R Company (see Problem [2.3](../../02%20Chapter%202%20Frequency%20Distributions/02%20Solved%20Problems/02%20Frequency%20Distributions,%20Histograms,%20and%20Frequency%20Polygons.ipynb#Problem-2-3)).

### Solution 3-30
Here $N=65$ and $N/2=32.5$. Since the sum of the first two and first three class frequencies and $8+10=18$ and $8+10+16=34$, respectively, the median class is the third class. Using the formula,

\begin{align}
\text{Median}=L_1+\left(\frac{N/2-(\sum{f})_1}{f_{\text{median}}}\right)c=\$269.995+\left(\frac{32.5-18}{16}\right)(\$10.00)=\$279.06
\end{align}

In [22]:
load('../../Saves//Table_2.6.Rdata')
N <- sum(Table.2.6[,1])
N.2 <- N/2
idx <- 1
while(sum(Table.2.6[1:idx,1]) < N.2){
    idx <- idx+1
}
f.sum.1 <- sum(Table.2.6[1:idx-1,1])
f.median <- as.vector(Table.2.6[idx,1])
round(269.995+(N.2-f.sum.1)/f.median*10,2)

[1] 279.06